vendor/DOtherSide/lib/include/DOtherSide/Status/UrlSchemeEvent.h

```diff
@@ -1,20 +0,0 @@
-#ifndef STATUS_URL_SCHEME_EVENT_H
-#define STATUS_URL_SCHEME_EVENT_H
-
-#include <QObject>
-
-namespace Status
-{
-    class UrlSchemeEvent : public QObject
-    {
-        Q_OBJECT
-
-        protected:
-            bool eventFilter(QObject* obj, QEvent* event) override;
-
-        signals:
-            void urlActivated(const QString& url);
```

Comment:
Not sure how it's intended to work. Will `urls_manager.nim` now listen to the window?

Reply:
@micieslak any ideas appreciated :) I had some creative ideas of routing the signal thru the QML layer but I don't really like this. Also I feel like StatusQ is not the right place for such an event filter